app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,43 @@
+class RecipesController < ApplicationController
+  def index
+    @user = User.find(current_user.id)
+    @recipes = Recipe.where(user_id: @user.id)
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+    @recipe_foods = @recipe.recipe_foods
+  end
```

Comment:
- [ ] Kindly fix the `n+1` query that happens in that action `Recipes#show` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Make%20sure%20there%20are%20no%20N%2B1%20queries%20happening.).